clc/config/config.go

```diff
@@ -47,7 +47,7 @@ func MakeHzConfig(props plug.ReadOnlyProperties, lg log.Logger) (hazelcast.Confi
 	wd := filepath.Dir(props.GetString(clc.PropertyConfig))
 	var cfg hazelcast.Config
 	cfg.Logger.CustomLogger = lg
-	cfg.Cluster.Unisocket = true
+	cfg.Cluster.Unisocket = false
```

Comment:
Is this intentional, if it is what is the reason I wonder?

Reply:
Tests are failing because of this change.

Reply:
It is intentional since we need to connect to all members to be able to timedMemberState. I will fix the tests